packages/protocol/test/AirnodeParameterStore.js

```diff
@@ -1,10 +1,10 @@
 /* globals context ethers */
```

Comment:
This file should have been deleted because the contract doesn't exist anymore

Reply:
Already removed in https://github.com/api3dao/airnode/pull/532